ui/src/app/shared/components/version-info/version-info-panel.tsx

```diff
@@ -42,14 +42,13 @@ export class VersionPanel extends React.Component<VersionPanelProps, {copyState:
         const formattedVersion = {
             'Argo CD': version.Version,
             'Build Date': version.BuildDate,
-            'Go': version.GoVersion,
-            'Compiler': version.Compiler,
+            'Go Version': version.GoVersion,
+            'Go Compiler': version.Compiler,
             'Platform': version.Platform,
             'ksonnet': version.KsonnetVersion,
             'jsonnet': version.JsonnetVersion,
             'kustomize': version.KustomizeVersion,
-            'Helm': version.HelmVersion,
-            'kubectl': version.KubectlVersion
```

Comment:
Kubectl version is still required.

Reply:
see https://github.com/argoproj/argo-cd/issues/5325#issuecomment-768093326

Reply:
will revert back, can go in another PR